packages/nx/src/project-graph/file-utils.ts

```diff
@@ -112,17 +117,29 @@ function defaultReadFileAtRevision(
   }
 }
 
+/**
+ * @deprecated This will be removed in v15, use project graph instead.
```

Comment:
I believe this is used by storbybook integration, so we won't be able to remove it. I think we might have to keep it.

Reply:
I **_think_** that any of the methods I marked as deprecated are no longer used by internal code, I'd have to double check that but am going to be away from my computer for the next 2 weeks.

Some of them are marked deprecated not necessarily to be removed, be just to discourage usage and point to project graph instead.